content/content.index.php

```diff
@@ -14,10 +14,17 @@ public function __viewIndex() {
 		$this->addStylesheetToHead(URL . '/extensions/dashboard/assets/dashboard.index.css', 'screen', 29422);
 		$this->addScriptToHead(URL . '/extensions/dashboard/assets/dashboard.index.js', 29423);
 		
+		// https://github.com/symphonycms/symphony-2/wiki/Migration-Guide-to-2.5-for-Developers#properties
+		if (is_callable(array('Symphony', 'Author'))) {
+			$author = Symphony::Author();
+		} else {
+			$author = Administration::instance()->Author;
+		}
+
 		// Add welcome message
 		$hour = date('H');
 		$welcome = __('Nice to meet you');
-		if(Administration::instance()->Author->get('last_seen') != NULL) {
+		if($author instanceof Author && $author->get('last_seen') != NULL) {
```

Comment:
I think that `$author instanceof Author` does not have any purpose. I would also declare the $author variable properyly ($author = null, before the if)